be1-go/hub/election.go

```diff
@@ -87,6 +90,7 @@ func (c *laoChannel) createElection(msg message.Message) error {
 		data.EndTime,
 		false,
 		getAllQuestionsForElectionChannel(data.Questions),
+		getAllAttendees(c.attendees),
```

Comment:
Maybe you can just pass `c.attendees` to the struct and there is no need for the function `getAllAttendees`